src/o365/spfx/commands/project/project-upgrade.ts

```diff
@@ -488,6 +507,99 @@ ${f.resolution}
     return s.join('').trim();
   }
 
+  private getTourReport(findings: FindingToReport[]): string {
+    const tourFindings: FindingTour = {
+      title: "Upgrade Project",
+      steps: []
+    };
+
+    let addedBuildStep: boolean = false;
+    const buildStep: FindingTourStep = {
+      file: "./package.json",
+      title: "REQUIRED: Build project",
+      description: "### REQUIRED\r\n\r\nBuild the project\r\n>> gulp build",
```

Comment:
Hey @hugoabernier, we don't include `gulp build` in other reports. Should it be added everywhere or is it just specific to tour?

Reply:
It doesn't need to be there, but the tour is guiding the user through the process, and I wanted to make sure they know where to build, because the next steps are about cleanup